floss/language/go/extract.py

```diff
@@ -18,6 +18,50 @@
 MIN_STR_LEN = 6
 
 
+def calculate_coverage(enhanced_static_strings: List[StaticString], static_strings: List[StaticString]) -> float:
+    all_static_string_list = []
+
+    for string_obj in static_strings:
+        all_static_string_list.append(string_obj.string)
+
+    extracted_strings = []
+    total_chars_covered = 0
+
+    for j in enhanced_static_strings:
+        k = j.string
+        for string_obj in static_strings:
+            if k in string_obj.string:
```

Comment:
I think if found once, the respective string needs to removed from `static_strings`.